services/121-service/src/registration/mappers/registration-views.mapper.ts

```diff
@@ -0,0 +1,116 @@
+import { omit } from 'lodash';
+
+import {
+  RegistrationDataInfo,
+  RegistrationDataRelation,
+} from '@121-service/src/registration/dto/registration-data-relation.model';
+import { RegistrationAttributeDataEntity } from '@121-service/src/registration/registration-attribute-data.entity';
+import { RegistrationViewEntity } from '@121-service/src/registration/registration-view.entity';
+
+type RegistrationViewWithoutData = Omit<RegistrationViewEntity, 'data'>;
+
+export class RegistrationViewsMapper {
+  static mapRootRegistration({
```

Comment:
I'm wondering if these method names are descriptive enough. Typically with map I expect clarity on from what to what. To what are you mapping root registration here? I see the code, but can't really put a name on it. But if not, then 'map' is maybe not the right verb to use here? (and similar cases)